### PR TITLE
CS-28

### PR DESCRIPTION
**Pull request by Alvindelito**
## This PR:

- Creates the following files in the directory `./src/components/Scheduling`:
  - `ConfirmationScreen.js`
  - `ConfirmationStyles.js`
  - `ListSelectedDateAndTimes.js`
- Made changes to `Header.js` to get rid of the extra text leftover from `CalendarSelectedTimes.js` when there are no `action` props to list`

### Figma Link:
[https://www.figma.com/file/kwUcXHYpKMHhuktdElLmmw/Calendaring-Home-Unite-Us-Copy?node-id=154%3A736](url)

### JIRA Link to Ticket
[https://calendaringservice.atlassian.net/browse/CS-28?atlOrigin=eyJpIjoiM2NhZDRmNjg3NGQ0NDdjYWJhOTg0ZTA4MjA3ZmExZjAiLCJwIjoiaiJ9](url) 

### Notes
- I made a guess on what the dataset may look like in the hardcoded data, but it can be easily changed once we get the actual dataset from the API
- I added an `overflow-y` attribute to the page to allow it to be scrollable without conflicting with the navigation buttons at the bottom
- The styles were inconsistent in the Figma between the three mockups, so I based the styles on the `Host Preferences` Mockup.

### Screenshot
Scrolled all the way up
Screen Shot 2021-06-02 at 1 55 59 PM

Scrolled all the way down
Screen Shot 2021-06-02 at 1 56 10 PM

### CODE REVIEW

File: src/components/Scheduling/ConfirmationStyles.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    alignItems: 'center',
+  },
+  subheading: {
+    fontweight: 500,
+    color: '#595959',
+    letterSpacing: '1px',
+  },
+  editButton: {
+    fontSize: '18px',
+    color: '#249BE5',
```
Comment (jonathanbutler7) [inline src/components/Scheduling/ConfirmationStyles.js:26]: Are the hard coded colors (like `color: '#249BE5'`, and others in this file) not available in the theme?

**Review by jonathanbutler7 — CHANGES_REQUESTED** [review]
Code looks great. Just left you a couple of questions.

Could you also make sure to continue Kimberly's react-router work, so this page is accessible by clicking the next button on the previous screen?

File: src/components/Scheduling/ConfirmationStyles.js (hunk 1 above)
Comment (Alvindelito) [reply src/components/Scheduling/ConfirmationStyles.js:26]: Just realized that `#249BE5` is in the theme. I'll fix that shortly

All of the other colors are not in the theme file however

- `#595959`
- `#2F2F2F`

File: src/components/Scheduling/ConfirmationStyles.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    marginRight: '1rem',
+  },
+  day: {
+    letterSpacing: '1px',
+    fontWeight: 500,
+  },
+  date: {
+    letterSpacing: '1px',
```
[the remark this answers is not in the document]
Comment (Alvindelito) [reply src/components/Scheduling/ConfirmationStyles.js:54]: Sorry, I'm not too sure which property you're referencing.

File: src/components/Scheduling/ConfirmationStyles.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  buttonContainer: {
+    position: 'absolute',
+    bottom: 0,
+    right: 0,
+    margin: '1rem',
+  },
+  navButtons: {
+    letterSpacing: '1px',
+    textTransform: 'capitalize',
```
[the remark this answers is not in the document]
Comment (Alvindelito) [reply src/components/Scheduling/ConfirmationStyles.js:71]: Yes, the button text is all uppercase by default. It does switch back to uppercase once I remove the `textTransform`. I made it `capitalize` based on the figma mockup. Do you want me to remove the `textTransform` property?

File: src/components/Scheduling/ConfirmationStyles.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonathanbutler7) [reply src/components/Scheduling/ConfirmationStyles.js:54]: I'm referring to `letterSpacing: '1px',` which is used several times.

File: src/components/Scheduling/ConfirmationStyles.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (jonathanbutler7) [reply src/components/Scheduling/ConfirmationStyles.js:71]: Yes, if the property is not necessary, then please remove it.

File: src/components/Scheduling/ConfirmationStyles.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Alvindelito) [reply src/components/Scheduling/ConfirmationStyles.js]: I removed all of the duplicate `letterSpacing: '1px';` and condensed it into one class called `letterSpacing`, and passed that class to the relative elements

**Review by jonathanbutler7 — APPROVED** [review]
👍

**Pull request merged**